cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
@psav Just tested it on 5.6  with the delay set to 60 and it passed, I'll give it another spin on 5.7 right now, just to be sure.